docs/sources/oncall-api-reference/alertgroups.md

```diff
@@ -130,6 +130,34 @@ curl "{{API_URL}}/api/v1/alert_groups/I68T24C13IFW1/unresolve" \
 
 `POST {{API_URL}}/api/v1/alert_groups/<ALERT_GROUP_ID>/unresolve`
 
+## Silence an alert group
+
+```shell
+curl "{{API_URL}}/api/v1/alert_groups/I68T24C13IFW1/silence" \
+  --request POST \
+  --header "Authorization: meowmeowmeow"
+```
```

Comment:
This curl call is missing the delay param, right?

Reply:
yup I have Made the changes